src/diagram/components/quantity-node.tsx

```diff
@@ -126,8 +127,13 @@ const _QuantityNode: React.FC<IProps> = ({ data, isConnectable }) => {
   const targetNodeHandleStyle = {height: nodeHeight, width: nodeWidth, left: "1px", opacity: 0, borderRadius: 0};
   const sourceHandleStyle = {border: "1px solid white", borderRadius: "6px", width: "12px", height: "12px", background: "#bcbcbc"};
 
+  const classnames = classNames(
+    "node",
+    hasExpression && "expression-shown",
+    data.dqRoot.selectedNode === data.node && "selected"
+  );
```

Comment:
This is subjective, but using classNames' hash argument makes this a little more clear to me. I think this is approach is uses in some parts of CLUE.
```suggestion
  const classnames = classNames({
    node: true,
    "expression-shown": hasExpression,
    selected: data.dqRoot.selectedNode === data.node
  });
```